save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxInternalController.kt

```diff
@@ -20,15 +39,21 @@ typealias StringResponse = ResponseEntity<String>
  */
 @RestController
 @RequestMapping("/sandbox/internal")
-class SandboxInternalController {
+class SandboxInternalController(
+    private val storage: Storage<SandboxStorageKey>,
+    private val agentRepository: SandboxAgentRepository,
+    private val objectMapper: ObjectMapper,
+) {
     /**
      * @param agentVersion
      * @return Mono with empty body
      */
     @PostMapping("/saveAgentVersion")
     fun saveAdditionalData(
         @RequestBody agentVersion: AgentVersion
-    ): Mono<Unit> = Mono.empty()
+    ): Mono<Unit> = {
+        // Do nothing for now
```

Comment:
Todo function may be here?

Reply:
nope, it's normal implementation. no need to save agent version for sandbox mode